### ISSUE TITLE
Github flow - poor use of issues

### ISSUE THREAD

**Issue by majakudlicka** [FACG2/getGitHub#32]
It's a good practice to make use of Github issues when working on a project. You should reference the issue number is your commit messages so that you can easily see how certain features got built (history of commits against a certain issue)

**Comment by Walidmash** [comment]
yes , Kefah laptop has a problem in pushing files , we spend more than hour trying to fix it, but we couldn't , so kefah sent us her codes through gitter, 
and yes we will put this note in mind in next project,
thank you Maja :)

**Comment by Walidmash** [comment]
ok, we will start using this way instead of writing the notes on papers
thank you ^^
